registration-processor/stage-groups/group-1/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>io.mosip.registrationprocessor</groupId>
+		<artifactId>stage-groups</artifactId>
+		<version>1.2.0-SNAPSHOT</version>
+	</parent>
+	
+	<artifactId>group-1</artifactId>
```

Comment:
follow full name (registration-processor-stage-group-1) for all groups everywhere(folder names, maven artifact etc)

Reply:
Done.